documentation/modules/ref-kafka-user.adoc

```diff
@@ -41,12 +45,12 @@ spec:
   # ...
 ----
 
-When the user is created by the User Operator, it will create a new secret with the same name as the `KafkaUser` resource.
-The secret will contain a public and private key which should be used for the TLS Client Authentication.
-Bundled with them will be the public key of the client certification authority which was used to sign the user certificate.
-All keys will be in X509 format.
+When the user is created by the User Operator, it creates a new secret with the same name as the `KafkaUser` resource.
+The secret contains a public and private key to use for TLS client authentication.
+Bundled with the keys is the public key of the client Certificate Authority (CA) used to sign the user certificate.
+All keys are in X.509 format.
 
-.An example of the `Secret` with user credentials
+.An example `Secret` with user credentials
```

Comment:
The example seems to be missing the p12 file and password. We should add it. Please let me or @ppatierno know if you need the example.

Reply:
Thanks @scholzj . I was going to ask about P12. Example would be useful.

Reply:
The data section now looks something like this:

```yaml
data:
  ca.crt: # ...
  user.crt: # ...
  user.key: # ...
  user.p12: # ...
  user.password: # ...
```

Reply:
Updated:

> Secrets provide private keys and certificates in PEM and PKCS #12 formats.
> For more information on securing Kafka communication with Secrets, see xref:security-{context}[].
> 
> ```
> data:
>   ca.crt: # Public key of the client CA
>   user.crt: # User certificate that contains the public key of the user
>   user.key: # Private key of the user
>   user.p12: # PKCS #12 archive file for storing certificates and keys
>   user.password: # Password for protecting the PKCS #12 archive file.
> ```
>